pex/pex_builder.py

```diff
@@ -402,9 +441,14 @@ def add_distribution(self, dist, dist_name=None):
           this will be inferred from the distribution itself should it be formatted in a standard way.
         :type dist: :class:`pkg_resources.Distribution`
         """
+        if dist.location in self._distributions:
```

Comment:
The new `SYMLINK` `copy_mode` brought to light the fact we were doing over-writes here when populating the PEXBuilder with the results of multi-target resolves whenever a distribution applied to more than one target (say a universal wheel). This was a harmless time-waste previously.